### PR TITLE
Add Hello World in C++

### PR DESCRIPTION
**Pull request by AmodhShenoy**
This Pull Request resolves Issue #3 

Description:
In this Pull Request we have done the following additions/edits for the code:
We just output a message saying "Hello World"

### CODE REVIEW

File: C/cprinting.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include<iostream.h>
```
Comment (MJ10) [inline C/cprinting.c]: Change the filename to your github username

File: StringRev/StringRev.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include<iostream.h>
```
Comment (MJ10) [inline StringRev/StringRev.c]: Change the filename to your github username and move to a folder named `reverse`

File: Strngrev/StrngRev.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include<iostream.h>
```
Comment (MJ10) [inline Strngrev/StrngRev.c]: Delete this file, since it is a duplicate

File: C/cprinting.c (hunk 1 above)
Comment (AmodhShenoy) [reply C/cprinting.c]: ok

**Comment by salman-bhai** [comment]
Why close the PR?

**Comment by MJ10** [comment]
He made a new one with the changes. So I closed this one.

**Comment by salman-bhai** [comment]
Cool cool.

**Pull request closed without merging**